pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "ftp://ftp.mozilla.org/pub/thunderbird/releases/${verName}/source/thunderbird-${verName}.source.tar.bz2";
-    sha1 = "90e18f8ecccdaf1ee39493223a7e3ad8b3b7bede";
+    sha1 = "7bb0c85e889e397e53dcbcbd36957dbd7c8c10bd";
```

Comment:
Our fetchurl doesn't support sha512, and most likely nix itself doesn't either (for the fixed-output has format).